walk-in-interview/src/main/java/com/google/utils/ServletUtils.java

```diff
@@ -0,0 +1,140 @@
+package com.google.utils;
+
+import com.google.gson.Gson;
+
+import javax.servlet.http.HttpServletRequest;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/** Util methods related to servlet. */
+public final class ServletUtils {
+    private ServletUtils() {}
+
+    /**
+     * @return the value of parameter with the {@code name} in the {@code request}
+     *         or returns {@code defaultValue} if that parameter does not exist.
+     */
+    public static String getStringParameter(HttpServletRequest request, String name, String defaultValue) {
+        String value = request.getParameter(name).trim();
+        if (value == null || value.isEmpty()) {
+            return defaultValue;
+        }
+        return value;
+    }
+
+    /**
+     * Gets the integer parameter from html form input.
+     *
+     * @param request Http request.
+     * @param parameterName Input id in html element.
+     * @param defaultValue Default integer.
+     * @return Parsed integer or default value if exception occur
+     */
+    public static int getIntParameter(HttpServletRequest request, String parameterName, int defaultValue) {
+        String resultStr = request.getParameter(parameterName).trim();
+
+        try {
+            int result = Integer.parseInt(resultStr);
+
+            return result;
+        } catch (NumberFormatException e) {
+            // TODO(issue/12): error handling; can consider to add logging to log the error
+
```

Comment:
nit: remove newline. same for the other cases below

Reply:
Fixed